apps/scraper/forms.py

```diff
@@ -0,0 +1,12 @@
+from django import forms
+from .models import Site
+
+
+class ChoseSiteForm(forms.Form):
+    for site in range(len(Site.objects.all())):
+        locals()['site_' + str(site + 1)] = forms.BooleanField(
```

Comment:
Don't get used to using `locals()` for the reasons persuasively described here:
https://stackoverflow.com/a/1901720

Reply:
> Don't get used to using `locals()` for the reasons persuasively described here:
> https://stackoverflow.com/a/1901720

I know that. Unfortunately, we don't can use ModelForm because Site model don't use BoolenField. Untill I use locals forms was hard-code. I can return that if you want this.